### PR TITLE
Drop Python 2.7 support

### LINKED ISSUE #24: Pack fails to install on Python 3.8
The pack fails to install on Python 3.8.
I found the solution was to install `distro` before installing the package.

While I can add that to the requirements, should we possibly switch to a newer version of the `salt` module?
By moving to a new version of the module we would loose Python2.7 support.

I am happy to submit the PR either way.
Just want feed back before I submit the PR

### PR DESCRIPTION
**Pull request by bmarick**
PR to resolve #24

### CODE REVIEW

**Comment by fjudith** [comment]
Please merge this PR.
Just upgrade to stackstorm 3.7.0 on docker and it is not working since base image upgraded to Ubuntu 20.04 LTS (Focal Fossa) which runs Python v3.8

**Comment by bmarick** [comment]
> Please merge this PR. Just upgrade to stackstorm 3.7.0 on docker and it is not working since base image upgraded to Ubuntu 20.04 LTS (Focal Fossa) which runs Python v3.8

@fjudith  you can point at my repo for your pack until this is merged

**Comment by nzlosh** [comment]
Thank you for this PR @bmarick.

File: pack.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ keywords:
   - salt
   - cfg management
   - configuration management
-version: 1.0.1
+version: 2.0.1
```
Comment (arm4b) [inline pack.yaml:9]: The changelog was referring to `2.0.0`, while we actually released `2.0.1` here :man_shrugging:

**Pull request merged**